raiden/messages.py

```diff
@@ -472,12 +524,9 @@ def from_event(cls, event):
 
 @dataclass(repr=False, eq=False)
 class RevealSecret(SignedRetrieableMessage):
-    """Message used to reveal a secret to party known to have interest in it.
+    """Reveal the a lock's secret.
 
-    This message is not sufficient for state changes in the raiden Channel, the
-    reason is that a node participating in split transfer or in both mediated
-    transfer for an exchange might can reveal the secret to it's partners, but
-    that must not update the internal channel state.
+    This message is not sufficient to unlock a lock, vide Unlock.
```

Comment:
vide?

Reply:
https://www.thefreedictionary.com/vide

Reply:
oh wow ... I did not know that word either. Perhaps use a more common one?